docs/pages/ch00/basics/constructor.md

```diff
@@ -1,9 +1,24 @@
+---
+content:
+  horizontalPadding: 50px
+  width: 100%
+  verticalPadding: 30px
+---
+
```

Comment:
Is there a way to modify theme/page template to not have to add this styling header on each pages?

Reply:
Looked into this, couldn't find anything specific in docs regarding this. Will look more and update you